### ISSUE TITLE
Only first web push notification is shown

### ISSUE THREAD

**Issue by antochiw-monarch** [OneSignal/OneSignal-Website-SDK#857]
**Description:**

Only the first web push notification pops up. Further notifications only update the current one in the Notification Center, but the user is not alerted about the updates.

This is caused by the grouping of the notifications. Seems like the `appId` is used to `tag` the notification. Is there any way to provide a custom value to be used there, or to disable grouping at all?

**Steps to Reproduce Issue:**

I believe this can be reproduced with any web push notification:

1. Visit any website using OneSignal web push notifications
2. Trigger a web push notification, either form the dashboard or from the API
3. The first notification should be displayed in the corner, as expected
4. After the first notification slides away, trigger a second notification
5. The second notification would not appear on the screen. However, it will be there if you open the notification center.

**Anything else:**

**OS:** macOS 11.2.3
**Browser:** Chrome 93

In the `serviceWorker.ts`, on `displayNotification` method, if I comment out `tag` and `renotify` options, then the web push notifications show as expected.

```
       /*
       Tags are any string value that groups notifications together. Two
       or notifications sharing a tag replace each other.
       */
      tag: extra.tag,  // <---- removing the tag along with renotify shows notifications properly
      /*
       On Chrome 47+ (desktop), notifications will be dismissed after 20
       seconds unless requireInteraction is set to true. See:
       https://developers.google.com/web/updates/2015/10/notification-requireInteractiom
       */
      requireInteraction: extra.persistNotification,
      /*
       On Chrome 50+, by default notifications replacing
       identically-tagged notifications no longer vibrate/signal the user
       that a new notification has come in. This flag allows subsequent
       notifications to re-alert the user. See:
       https://developers.google.com/web/updates/2016/03/notifications
       */
      // renotify: true,  // <---- removing the tag along with renotify shows notifications properly
```

**Comment by vhd-luke** [comment]
Having the same problem, did you ever find a solution to this?

**Comment by rgomezp** [comment]
Howdy,
We use the appId as the `tag` value so the notifications coming from the same app would be grouped together. However, the renotify setting should override this and the notification should come in as expected. I find it strange that removing both is what fixed it for you.

Can you make sure you're on the latest chrome version? Perhaps there was an issue that has since been patched.

**Comment by rgomezp** [comment]
Closing for now. Please leave a comment if you are still seeing this issue and we can reopen

**Comment by katrinajaneczko** [comment]
Hi, I am still seeing this issue. I am on Version 112.0.5615.137. I have an existing PWA with react as the frontend. I merged my existing service worker and the OneSignal one using this article https://www.geekyhub.in/post/using-one-signal-with-react-js/ then I followed https://onesignal.com/blog/how-to-integrate-push-notifications-in-react/ and on my Mac it works in Safari, but in Chrome only the welcome notification is displayed. Then all subsequent notifications are not displayed. If I click the time to open the Notification Center, I see the notifications there, I am just not alerted to them coming in.
